svcomp-benchmark/results/README.md

```diff
@@ -0,0 +1,81 @@
+Data collected from running [RV-Match](https://runtimeverification.com/match) on the [SV-COMP Benchmark Verification Tasks](https://github.com/sosy-lab/sv-benchmarks). These results were collected on April 13th, 2016. More details about the verification tasks, and the structure of the benchmarks can be found at [SV-COMP's website](http://sv-comp.sosy-lab.org/2016/index.php).
+
```

Comment:
Is the above correct English?  I would say "can be found on the SV-COMP website".  Maybe @dwightguth can quickly revise the English?